stories/useNetworkState.story.mdx

```diff
@@ -0,0 +1,49 @@
+import { Example } from './useNetworkState.stories';
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title={'Sensor/useNetwork'} component={Example} />
+
+# useNetwork
+
+Tracks the state of browser's network connection.
+
+> As of the standard, it is not guaranteed that browser connected to the _Internet_, it only
```

Comment:
Maybe we should quote MDN here and add a link?

`while you can assume that the browser is offline when it returns a false value, you cannot assume that a true value necessarily means that the browser can access the internet`

https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/onLine